agent/internal/client/base_client.go

```diff
@@ -1058,3 +1058,18 @@ func (c *baseCommunicator) CreateInstallationToken(ctx context.Context, td TaskD
 
 	return token.Token, nil
 }
+
+// MarkFailedTaskToRestart will mark the task to be automatically restart upon completion
```

Comment:
nit: "to automatically restart" or "to be automatically restarted"

Reply:
Done.